docs/framework/guides/contributing/git-commit-message-convention.md

```diff
@@ -47,23 +47,33 @@ Each commit can contain additional notes which will be inserted into the changel
 | Type | Is backward compatible? |
 | --- | --- |
 | `NOTE` | Yes |
-| `BREAKING CHANGES` or `BREAKING CHANGE` | No |
+| `MAJOR BREAKING CHANGE`, `MINOR BREAKING CHANGE` or `BREAKING CHANGE` | No |
 
-If any visible change contains the `BREAKING CHANGE` note, the next release will be marked as `major` automatically.
+The `BREAKING CHANGE` is an alias to a `MAJOR BREAKING CHANGE`.
```

Comment:
I think we were supposed to remove `BREAKING CHANGE` as an unnecessary complication.

Reply:
Sorry, I considered that but I wrote in https://gist.github.com/Reinmar/eb12358e84caae87dda862eb438089bd#notesthat we'll still support it.